scripts/bundle-generation/validate-image-keys.py

```
@@ -0,0 +1,270 @@
+    Args:
+        operator_path: Path to operator repo
+        bundle_path: Path to bundle repo
+        version: Version string
+
+    Returns:
+        bool: True if validation passes, False otherwise
+    """
+    operator_name = Path(operator_path).name
+    bundle_name = Path(bundle_path).name
+
+    logging.info(f"Validating image keys:")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)